ad4630 fails dma_rx test

When running test_ad4630.py, dma_rx test failed with the following:

```
    def rx(self):
>       if not self._rx__rxbuf:
E       AttributeError: 'ad4630' object has no attribute '_rx__rxbuf'
```

I attached full test result.
[ad4630-test-result.txt](https://github.com/analogdevicesinc/pyadi-iio/files/14274783/ad4630-test-result.txt)

--- a/adi/ad4630.py
+++ b/adi/ad4630.py
@@ -63,12 +63,7 @@ class ad4630(rx, context_manager, attribute):
         rx.__init__(self)
 
     def rx(self):
-        if not self._rx__rxbuf:
-            self._rx_init_channels()
-        self._rx__rxbuf.refill()
-        buff = np.frombuffer(self._rx__rxbuf.read(), dtype=np.uint32)
-
-        data = [buff[0::2], buff[1::2]]
+        data = self._rx_buffered_data()
         temp = []
         if self._num_rx_channels != 2:
             for ch in range(0, self._num_rx_channels):

